src/components/NavMenu.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class NavMenu extends Component {
+  constructor(props) {
+    super(props);
+    const { location: { pathname = '' } = {} } = window;
```

Comment:
fix to use react-router...how do make a PR with code you know you need to change later?